pkg/cpi/mockingUtils.go

```diff
@@ -45,3 +63,18 @@ func GetEmptyHTTPResponseBody() (*http.Response, error) {
 	}
 	return &res, nil
 }
+
+//GetNegativeCaseHTTPResponseBody -Negative case http respose body
+func GetNegativeCaseHTTPResponseBody() (*http.Response, error) {
```

Comment:
I don't line that this returns an error `nil`, **always**. Why not only return the `*http.Response` and where this function is used you return the `nil` error?